lib/rspec/api_doc/strings.rb

```diff
@@ -1,12 +1,10 @@
-require 'uri'
-
 module RSpec
   module ApiDoc
     module Strings
     module_function
 
       def pretty_escape(text, sep = '-')
-        URI.escape(text.strip.gsub(/[[:space:]]+/, sep))
+        CGI.escape(text.strip.gsub(/[[:space:]]+/, sep))
```

Comment:
It looks like the behavior of `CGI.escape` is different, but that's probably okay?


```rb
pry(main)> CGI.escape("http://example.com/")
=> "http%3A%2F%2Fexample.com%2F"
pry(main)> URI.escape("http://example.com/")
(pry):8: warning: URI.escape is obsolete
=> "http://example.com/"
```